setup.py

```diff
@@ -65,6 +65,7 @@ def main():
         "attrs>=17.4.0",
         "more-itertools>=4.0.0",
         "atomicwrites>=1.0",
+        "monotonic",
```

Comment:
I left it this way because then you can access monotonic consistently via `monotonic.monotonic()`, though sure, at this point we only use it for one thing and only in py2 (and it seems is going away anyways).